src/Libraries/CoreNodeModels/ColorRange.cs

```diff
@@ -32,6 +32,15 @@ namespace CoreNodeModels
     [AlsoKnownAs("DSCoreNodesUI.ColorRange")]
     public class ColorRange : NodeModel
     {
+        private List<Color> _defaultColors;
```

Comment:
in general I think we avoid the use of this naming pattern:
https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards

Reply:
renamed